test/src/TapedTest.cpp

```diff
@@ -0,0 +1,58 @@
+#include <helpers/commonHelpers.hpp>
+#include <rgl/api/extensions/tape.h>
+
+#include <filesystem>
+#include <spdlog/fmt/ostr.h>
+
+#if RGL_BUILD_PCL_EXTENSION
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+#include <pcl/io/pcd_io.h>
+#endif
+
+class TapedTest : public RGLTest
+{
+protected:
+	const std::string benchmarkDataDirEnvVariableName = "RGL_TAPED_TEST_DATA_DIR";
+	const float epsilon = 1e-2f;
+};
+
+TEST_F(TapedTest, compare_pcd_files)
+{
+#if RGL_BUILD_PCL_EXTENSION
+
+	// It is necessary to set environment variable on benchmark data directory
+	if (std::getenv(benchmarkDataDirEnvVariableName.c_str()) == nullptr) {
+		const std::string msg = fmt::format(
+		    "Skipping Taped Test - benchmark data directory must be provided in environment variable '{}'",
+		    benchmarkDataDirEnvVariableName);
+		GTEST_SKIP() << msg;
+	}
+
+	const std::string benchmarkDataDir = std::getenv(benchmarkDataDirEnvVariableName.c_str());
+	const std::string testTapePath{(std::filesystem::path(benchmarkDataDir) / "awsim-mesh2pcd").string()};
+	const std::string expectedOutputPath{
+	    (std::filesystem::path(benchmarkDataDir) / "expected-output" / "awsim-mesh2pcd.pcd").string()};
+	const std::string outputPath{(std::filesystem::current_path() / "output.pcd").string()};
```

Comment:
I'd avoid hardcoding filenames (e.g. `awsim-mesh2pcd`). Constexpr variables at the top of the test would improve readability and simplify test modifications if names would change.